addons/payment_razorpay_oauth/views/razorpay_error_template.xml

```diff
@@ -6,18 +6,16 @@
             - 'error_message' - The reason of the error.
             - 'redirect_url' - The URL to the Razorpay account.
         -->
-        <t t-call="portal.frontend_layout">
```

Comment:
Using Frontend layout gives error. may be same error happening in Amazon as we taken motivation from Amazon to add this template

Reply:
I fixed the error in Amazon with https://github.com/odoo/enterprise/pull/71987. I'll apply the same fix to Razorpay.